src/a4/domain/Street.java

```diff
@@ -1,5 +1,54 @@
 package a4.domain;
 
-public class Street {
+public class Street extends Property {
+	int houseCount;
+	int hotelCount;
+	String color;
 
+	Street(String name, int value, String color) {
+		super(name, value);
+		this.color = color;
+	}
+
+	public int getHouseCount() {
+		return houseCount;
+	}
+
+	public void setHouseCount(int houseCount) {
+		this.houseCount = houseCount;
+	}
+
+	public int getHotelCount() {
+		return hotelCount;
+	}
+
+	public void setHotelCount(int hotelCount) {
+		this.hotelCount = hotelCount;
+	}
+
+	public String getColor() {
+		return color;
+	}
+
+	public void setColor(String color) {
+		this.color = color;
+	}
+
+	public void addHouse() {
+		if (houseCount < 4)
+			houseCount++;
+		else if (hotelCount == 0) {
+			houseCount = 0;
+			hotelCount = 1;
+		}
```

Comment:
So the most that you can have on a property is 1 hotel which is equal to 5 houses? Monoply is an interesting game jaja